app/views/devise/sessions/new.html.erb

```diff
@@ -1,29 +1,29 @@
 <body class="login-view">
   <div class="login">
-    <h2 class="login_label">Log in</h2>
+    <h2 class="login_label">Log in.</h2>
 
     <%= form_for(resource, as: resource_name, url: session_path(resource_name), class:"log-in-form") do |f| %>
 
       <div class="lo-form-content">
         <div class="lo_field">
-          <%= f.label :email %><br />
-          <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+          <%= f.label :email, class:"lo_lbl" %>
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"form-control", type:"email", placeholder:"Enter email" %>
         </div>
 
         <div class="lo_field">
-          <%= f.label :password %><br />
-          <%= f.password_field :password, autocomplete: "current-password" %>
+          <%= f.label :password, class:"lo_lbl" %>
+          <%= f.password_field :password, autocomplete: "current-password", class:"form-control", type:"password", placeholder:"Password" %>
         </div>
 
         <% if devise_mapping.rememberable? %>
-          <div class="lo_field">
-            <%= f.check_box :remember_me %>
-            <%= f.label :remember_me %>
+          <div class="form-group form-check">
+            <%= f.check_box :remember_me, class:"form-check-input", type:"checkbox" %>
+            <%= f.label :remember_me, class:"form-check-label" %>
```

Comment:
nice use of classes / ids. this should make it easier to make cucumber scenarios that check within certain html elements on the page. Good stuff!